canonical/ubuntu.com
# Setup playwright tests

## Done
- Set up playwright and write one simple test.
## QA

- Save [this credentials](https://pastebin.canonical.com/p/r3Cf49554s) in `.env.local`
- Command `dotrun`
- Command `yarn playwright test --ui` or `yarn playwright test`
- Check tests pass locally or in github checks

## Issue / Card

Fixes 
#https://warthogs.atlassian.net/jira/software/c/projects/WD/boards/801?selectedIssue=WD-7475
#https://warthogs.atlassian.net/jira/software/c/projects/WD/boards/801?selectedIssue=WD-7530
